expression/builtin_time_vec_test.go

```diff
@@ -52,9 +52,11 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.CurrentDate: {
 		{retEvalType: types.ETDatetime},
 	},
-	ast.MakeDate:   {},
-	ast.MakeTime:   {},
-	ast.PeriodAdd:  {},
+	ast.MakeDate: {},
+	ast.MakeTime: {},
+	ast.PeriodAdd: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt, types.ETInt}, geners: []dataGenerator{&rangeInt64Gener{1, 13}, &rangeInt64Gener{1, 20}}},
```

Comment:
Could you make a new `gener` reflecting the format `YYMM` or `YYYYMM` in builtin_time_vec_test.go?